src/expressions/operations.py

```diff
@@ -42,6 +56,14 @@ def __repr__(self):
     
     def expand(self):
         return Difference(self.exp1.expand(), self.exp2.expand())
+    
+    def contains(self, var):
+        if self.exp1.contains(var):
```

Comment:
Instead of doing: 
```
if exp1.contains(var) :
  return True
elif exp2.contains(var) :
  return True

```
It is simpler and cleaner to do 
```
if exp1.contains(var) or exp2.contains(var):
  return True
```